apps/budgeting/dashboard.py

```diff
@@ -14,9 +14,8 @@ class ExportBudgetingComponent(DashboardComponent):
     label = _("Export Excel")
 
     def is_effective(self, module):
-        module_app = module.phases[0].content().app
         return (
-            module_app == "a4_candy_budgeting"
+            module.blueprint_type == "PB"
```

Comment:
dont we need all budgeting types here?

Reply:
I think in aplus there is one phase PB available? at least in the Blueprint list I cannot find another one